flang/lib/FrontendTool/ExecuteCompilerInvocation.cpp

```diff
@@ -32,6 +32,10 @@ static std::unique_ptr<FrontendAction> CreateFrontendBaseAction(
     return std::make_unique<PrintPreprocessedAction>();
   case ParseSyntaxOnly:
     return std::make_unique<ParseSyntaxOnlyAction>();
+    break;
+  case EmitFir:
+    return std::make_unique<EmitFirAction>();
+    break;
```

Comment:
You don't need these unreachable `break` commands.

Reply:
Completely missed that, will update shortly, thank you!